tardis/visualization/tools/tests/test_sdec_plot.py

```diff
@@ -88,33 +101,50 @@ def sdec_ref_data_path(tardis_ref_path):
 class TestSDECPlotter:
```

Comment:
Can the `sdec_ref_data_path` function be removed now?

Reply:
Fixed in [`257931c` (#2785)](https://github.com/tardis-sn/tardis/pull/2785/commits/257931c30df41e9a13544c0949e0d686f510a5d3)